### DISCUSSION TITLE
Thoughts about creating DNS CAA records when issuing certificates ?

### DISCUSSION THREAD

**Discussion by kevinkknorr** [polymind-inc/acmebot#544] [Ideas]
I was thinking today that with the advent of CAA records it might be a nice feature to be able to add or update these when a certificate is issued.

**Comment by shibayan** [comment]
I do not believe that additional updates from Acmebot are a good idea since CAA records are intended to prevent certificates from being issued by unintended issuers.

**Comment by kevinkknorr** [comment]
That makes perfect sense !
